src/ui/common/rich-text-editor/EditorToolbar.js

```diff
@@ -0,0 +1,254 @@
+    <path className="ql-fill" d="M9.707,13l1.146-1.146a0.5,0.5,0,0,0-.707-0.707L9,12.293,7.854,11.146a0.5,0.5,0,0,0-.707.707L8.293,13,7.146,14.146a0.5,0.5,0,1,0,.707.707L9,13.707l1.146,1.146a0.5,0.5,0,0,0,.707-0.707Z" />
+  </svg>
+);
+
+const tableDeleteIcon = (
+  <svg viewBox="0 0 18 18">
+    <path className="ql-fill" d="M15.707,7l1.146-1.146a0.5,0.5,0,1,0-.707-0.707L15,6.293,13.854,5.146a0.5,0.5,0,0,0-.707.707L14.293,7,13.146,8.146a0.5,0.5,0,1,0,.707.707L15,7.707l1.146,1.146a0.5,0.5,0,1,0,.707-0.707Z" />
+    <path className="ql-fill" d="M6,5H3A1,1,0,0,0,2,6V8A1,1,0,0,0,3,9H6V5Z" />
+    <path className="ql-fill" d="M10,5H7V9h3a1,1,0,0,0,1-1V6A1,1,0,0,0,10,5Z" />
+    <g className="ql-fill ql-transparent">
+      <path d="M8,11h4V9a1,1,0,0,0-1-1H8v3Z" />
+      <path d="M7,11V8H4A1,1,0,0,0,3,9v2H7Z" />
+      <path d="M7,12H3v2a1,1,0,0,0,1,1H7V12Z" />
+      <path d="M8,12v3h3a1,1,0,0,0,1-1V12H8Z" />
+      <path d="M8,6h3a1,1,0,0,0,1-1V3a1,1,0,0,0-1-1H8V6Z" />
+      <path d="M4,6H7V2H4A1,1,0,0,0,3,3V5A1,1,0,0,0,4,6Z" />
+    </g>
+  </svg>
+);
+
+const hrIcon = (
+  <svg viewBox="0 0 18 18">
+    <path className="ql-fill" d="M15,12v2a.99942.99942,0,0,1-1,1H4a.99942.99942,0,0,1-1-1V12a1,1,0,0,1,2,0v1h8V12a1,1,0,0,1,2,0ZM14,3H4A.99942.99942,0,0,0,3,4V6A1,1,0,0,0,5,6V5h8V6a1,1,0,0,0,2,0V4A.99942.99942,0,0,0,14,3Z" />
+    <path className="ql-fill" d="M15,10H3A1,1,0,0,1,3,8H15a1,1,0,0,1,0,2Z" />
+  </svg>
+);
+
+const maximizeIcon = (
+  <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="expand-arrows-alt" className="svg-inline--fa fa-expand-arrows-alt fa-w-14" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512">
+    <path fill="currentColor" d="M448 344v112a23.94 23.94 0 0 1-24 24H312c-21.39 0-32.09-25.9-17-41l36.2-36.2L224 295.6 116.77 402.9 153 439c15.09 15.1 4.39 41-17 41H24a23.94 23.94 0 0 1-24-24V344c0-21.4 25.89-32.1 41-17l36.19 36.2L184.46 256 77.18 148.7 41 185c-15.1 15.1-41 4.4-41-17V56a23.94 23.94 0 0 1 24-24h112c21.39 0 32.09 25.9 17 41l-36.2 36.2L224 216.4l107.23-107.3L295 73c-15.09-15.1-4.39-41 17-41h112a23.94 23.94 0 0 1 24 24v112c0 21.4-25.89 32.1-41 17l-36.19-36.2L263.54 256l107.28 107.3L407 327.1c15.1-15.2 41-4.5 41 16.9z" />
+  </svg>
+);
+
+const ToolbarGroup = ({ children, className }) => (
+  <span
+    className={compact(['ql-formats', className]).join(' ')}
+  >
+    {children}
+  </span>
+);
+
+ToolbarGroup.propTypes = {
+  children: PropTypes.node.isRequired,
+  className: PropTypes.string,
+};
+
+ToolbarGroup.defaultProps = {
+  className: '',
+};
+
+const renderToolbarButton = (format, value, icon, tooltipText) => (
+  <button className={`ql-${format}`} value={value} type="button" title={tooltipText}>
+    {icon}
+  </button>
+);
+
+const EditorToolbar = ({ name, extraOptions }) => (
+  <div id={name} style={{ borderBottom: 'none' }}>
+    <ToolbarGroup>
+      {renderToolbarButton('history', 'undo', undoIcon, 'Undo (Ctrl + Z)')}
```

Comment:
do we want to locale these labels?

Reply:
I'd suggest to localize those labels, as well as the RTE files in CMS